calculator/src/Components/Result.js

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from "react";
+
+class ResultComponent extends Component {
+  render() {
+    let { result } = this.props;
+
+    return (
+      <div className="result">
+        <p>{result}</p>
```

Comment:
This could be a great place to have 

1.  a ternary to say if result is an empty string, display 0
2. look up toLocaleString() so you can get commas in your numbers